File: 2966. Divide Array Into Arrays With Max Difference/submission.ts
function divideArray(nums: number[], k: number): number[][] {
    if(nums.length % 3 != 0) return []
    nums.sort((a, b) => a - b)
    const output: number[][] = []
    for(var i = 0; i < nums.length; i += 3) {
        if(nums[i + 2] - nums[i] > k) return []
        output.push([nums[i], nums[i + 1], nums[i + 2]])
    }
    return output
};

File: 1291. Sequential Digits/submission.cs
public class Solution {
    public IList<int> SequentialDigits(int low, int high) {
        string NUMS = "123456789";
        int lowNumDigits = low.ToString().Length;
        int highNumDigits = high.ToString().Length;
        IList<int> output = new List<int>();
        
        for(int numDigits = lowNumDigits; numDigits <= highNumDigits; numDigits++) {
            bool complete = false;

            // iterations = (length + 1) - size
            for(int index = 0; index < (NUMS.Length + 1) - numDigits && !complete; index++) {
                int num = Int32.Parse(NUMS.Substring(index, numDigits));
                if(low <= num && num <= high) {
                    output.Add(num);
                }
                else if(num > high) {
                    complete = true;
                }
            }
        }
        return output;
    }
}